src/wrapper.ts

```diff
@@ -32,7 +32,12 @@ let isTraceInitialized = false;
 const externalInstrumentations = [];
 
 function requireIfAvailable(names: string[]) {
-  names.forEach((name) => safeRequire(name));
+  const instrumentedValues = new Set<string>();
+  names.forEach((name) => {
+    const required = safeRequire(name);
```

Comment:
This thing has always smelled to me. Isn't there maybe an API in Node.js that works like Python's `pks_resource.getDistribution()` https://setuptools.pypa.io/en/latest/pkg_resources.html ?

Reply:
There is no easy way to check if a package is installed most use try catch wich is even uglyer